trusted-types/block-string-assignment-to-Element-setAttributeNode.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<meta http-equiv="Content-Security-Policy" content="require-trusted-types-for 'script';">
+<title>trusted-types (TT): setAttributeNode with node from non-TT realm</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="nonSVGTestElements">
+    <iframe srcdoc="v"></iframe>
+    <embed src="v">
```

Comment:
For `embed` and `object` the expectation needs to be different right, per recent events?

Reply:
Presumably you refer to https://github.com/w3c/trusted-types/pull/486? Yes. Presumably more tests will have to be adapted; it seems cleaner to do that collectively in https://github.com/w3c/trusted-types/pull/486.

Reply:
Yeah I'm planning to go through the test suite and probably just removing most references to embed and object. With potentially one left to make sure that they're not enforcing TT restrictions.

So feel free to just not add them to this test, or to add them and I can remove them them in my PR.

Reply:
Just to loop back here I've started https://github.com/WebKit/WebKit/pull/26554

So it would be best if no new usages of embed or object were added.